test/screenshot/spec/mdc-drawer/fixture.scss

```diff
@@ -35,14 +42,38 @@ $custom-drawer-color: $material-color-orange-900;
   line-height: 1;
 }
 
-.custom-drawer--fill-color {
-  @include mdc-drawer-fill-color($custom-drawer-color);
+body {
+  display: flex;
+  height: 100vh;
 }
 
-.custom-drawer--fill-color-accessible {
-  @include mdc-drawer-fill-color-accessible($custom-drawer-color);
+.test-drawer-app-content {
+  flex: auto;
+  overflow: auto;
 }
 
-.custom-drawer--ink-color {
-  @include mdc-drawer-ink-color($custom-drawer-color);
+.test-main-content {
+  overflow: auto;
+  height: 100%;
+  padding-left: 16px;
+  padding-right: 16px;
 }
+
+.custom-drawer--fill-color {
```

Comment:
Typically we aim for each screenshot test class to apply exactly one mixin. Thus I'd aim for these to only apply the relevant mixin, as they were doing before. We can add more screenshot test pages for the other mixins later.